package.json

```
@@ -1,24 +1,25 @@
 {
   "name": "ap_desktop",
-  "productName": "AP Desktop",
+  "productName": "ap_desktop",
   "version": "1.3.0",
-  "description": "Graphical user interface for QUT's AnalysisPrograms.exe (AP.exe).",
+  "description": "Graphical user interface for QUT's AnalysisPrograms.exe (AP.exe) software.",
   "main": "src/index.js",
   "scripts": {
-    "start": "concurrently --kill-others \"npm run babel:async\" \"electron-forge start\"",
-    "package": "electron-packager . AP-Desktop --all --out=out/",
-    "make": "electron-forge make",
-    "dependencies": "chmod +x ./linux_dependencies.sh && ./linux_dependencies.sh",
-    "ap_download": "echo Downloading latest version of AP for Release. This could take up to 15 minutes.&& pwsh -ExecutionPolicy ByPass -File ./script_ap.ps1",
+    "ap_download": "pwsh -ExecutionPolicy ByPass -File ./scripts/script_ap.ps1",
+    "babel:async": "npx babel ./src/js/ --out-dir ./src/js-compiled/ -w",
+    "babel": "npx babel ./src/js/ --out-dir ./src/js-compiled/ && npx babel ./tests/test/ --out-dir ./tests/test-compiled/",
+    "clean": "rimraf out",
+    "dependencies": "chmod +x ./scripts/linux_dependencies.sh && ./scripts/linux_dependencies.sh",
     "install": "npm run ap_download && npm run babel",
-    "release": "electron-forge make",
-    "publish": "electron-forge publish",
+    "start": "concurrently --kill-others \"npm run babel:async\" \"electron-forge start\"",
     "test": "npm run babel && mocha tests/test-compiled/tests.js",
-    "babel": "npx babel ./src/js/ --out-dir ./src/js-compiled/ && npx babel ./tests/test/ --out-dir ./tests/test-compiled/",
-    "babel:async": "npx babel ./src/js/ --out-dir ./src/js-compiled/ -w"
+    "package": "electron-forge package",
+    "make": "npm run clean && electron-forge make",
+    "publish": "electron-forge publish",
+    "lint": "eslint src --color"
   },
   "keywords": [],
-  "author": "Charles Alleman",
+  "author": "Queensland University of Technology",
```

```suggestion
  "author": "QUT Ecoacoustics",
```

You should add yourself to the contributors field